src/main/java/vimification/model/DisplayTaskList.java

```diff
@@ -0,0 +1,55 @@
+package vimification.model;
+
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import vimification.model.task.Task;
+
+public class DisplayTaskList {
```

Comment:
Very nice - at good wrapper - make it symmetric with the `LogicTaskList`.

Reply:
I think as discussed, we are no longer using DisplayTaskList. We would encapsulate the ReadOnly barrier using a method of LogicTaskList instead (and the method will return an ObservableList<Task>)

Reply:
That's true, the wrapper is not necessary at the moment. But keep this one.